main.py

```diff
@@ -429,7 +429,7 @@ def main():
             "hausnummer": hausnummer,
             "plz": wohnort_plz,
             "ort": wohnort,
-            "phone": "+49" + str(telefonnummer).removeprefix("+49").removeprefix("0"),
+            "phone": f"+49{telefonnummer}",
```

Comment:
Das hier würde ich nicht ändern. So wies war ists besser, der Input change in Zeile `421` kann bleiben.

Reply:
Habs wieder reingenommen 👍